contributing/topics/guide-breaking-changes.md

```diff
@@ -172,16 +172,22 @@ The following example follows a fictional resource that will have the following
    
       if !features.FivePointOh() {
          args["enable_scaling"] = &pluginsdk.Schema{
-            Type:     pluginsdk.TypeBool,
-            Optional: true,
-            Computed: true,
-            Default:  false,
+            Type:          pluginsdk.TypeBool,
+            Optional:      true,
+            Computed:      true,
+            Default:       false,
             ConflictsWith: []string{"scaling_enabled"},
+            Deprecated:    "`enable_scaling` has been deprecated in favour of `scaling_enabled` and will be removed in v5.0 of the AzureRM Provider",
          }
          // When renaming a property both properties need to have `Computed` set on them until the old property is removed in the next major release
          // We also need to remember to set ConflictsWith on both the old and the renamed property to ensure users don't set both in their config
```

Comment:
I personally think that if only 1-2 attributes of a property's schema definition needs to be patched then it is acceptable to only overwrite those specific attributes instead of overwriting the whole schema definition for the property since it doesn't negatively impact the clean up effort and it is less lines of code.

But I understand that having exceptions like this increase the likelihood of variations entering the code and contributors/reviewers would need to remember this exception to the rule so I am in agreement with making this change.

I'd suggest extending this comment, or adding a note above/below this code snippet saying that even if only one attribute of a property's schema definition needs to be modified we should still be patching the whole schema definition.